README reorganise (#359)
UKGovernmentBEIS/beis-opss-psd: README.md

## Getting set up

See [getting set up](doc/getting-set-up.md).

## Contributing

See [contributing](CONTRIBUTING.md).

## Amazon Web Services

> We're using AWS for file storage on the S3 service. You'll need an account - ask someone on the team to invite you. If you get an error saying you don't have permission to set something, make sure you have MFA set up.

Reviewer: ```suggestion
We're using AWS for file storage on the S3 service. You'll need AWS account credentials.
```
